src/features/content-answers/body/ContentAnswersGridCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import BoxAi from '../../../icon/line/BoxAi';
+import Media from '../../../components/media';
+
+import './ContentAnswersGridCard.scss';
+
+type Props = {
+    children: React.ReactNode;
+};
+
+const ContentAnswersGridCard = ({ children }: Props) => {
+    if (!children) {
+        return null;
+    }
```

Comment:
is it possible for card to be called without children, its listed as required in the props?

Reply:
I think you are right! Removed this null condition check.